routes/web.php

```diff
@@ -14,3 +14,27 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Auth::routes();
```

Comment:
Where is the usage of https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=339395368 Route Parameters

Reply:
I thought it would be in feature edit blog or edit user.